go/src/koding/klient/machine/mount/index/kite.go

```diff
@@ -0,0 +1,55 @@
+package index
+
+import (
+	"github.com/koding/kite"
+)
+
+// KiteHandlerHead creates a kite handler function that, when called, invokes
+// index package Head method.
+func KiteHandlerHead() kite.HandlerFunc {
```

Comment:
I'm thinking whether having single `kiteHandler` would be more economical (if we're going to have 10+ kite methods then probably so) in terms of loc, something like:

```go
func KiteHandlerHead() kite.HandlerFunc {
    return kiteHandler(func(*Request)(interface{}, error) { return Head() })
}
```

🤔